### PR TITLE
MM² Icon

### PR DESCRIPTION
**Pull request by bugsounet**
* Create `MM²` icon
 * Allow to change default electron icon to this icon

### CODE REVIEW

File: MM².png
Comment (rejas) [inline MM².png:1]: nice idea. but can you name the file jsut mm2.png without the high 2 ? dont want to risk the spelling to condfuse some filesystems :-)

**Review by rejas — CHANGES_REQUESTED** [review]

**Comment by KristjanESPERANTO** [comment]
Out of curiosity: When and where is the icon displayed?

**Comment by bugsounet** [comment]
@rejas: wait, I will change "MM" color because it's so darkness... we see nothing lol

**Comment by bugsounet** [comment]
@KristjanESPERANTO 

something like this ?

image

and task bar (when minified or when multi-screen using)
image

**Comment by bugsounet** [comment]
@veeck, it's ok (i think)

**Review by rejas — APPROVED** [review]

**Comment by khassel** [comment]
so this is the new favicon when running under electron (which is only visible if not running fullscreen), but for server only we have another favicon in `index.html`. So should we use the same icon for both?

**Comment by rejas** [comment]
that index.html favicon is this base64 encoded small string?

**Comment by khassel** [comment]
I think so, displayed is 
grafik

**Comment by khassel** [comment]
you can replace this line e.g. with

```html
<link rel="icon" type="image/png" href="path/file.png">
```

to test other icons (the path must served by mm)

**Comment by bugsounet** [comment]
will not works `mm2.png` have no entry on express

**Comment by khassel** [comment]
that was meant with "the path must served by mm", for testing is must be copied to e.g. config or modules or ...

**Comment by bugsounet** [comment]
add an  `app.get` to serve `mm2.png` in server.js ;)
I'm in lazy mode :D

**Comment by rejas** [comment]
unifying them would make sense. and maybe also ask in the forum if we want to make a "design the mm2 logo" contest (no offence against your design, bugsounet, but we are developers, no designers :-)

**Comment by bugsounet** [comment]
sure it's an first approach... I have planned to do that in next version of MM²

**Pull request merged**